tests/conftest.py

```diff
@@ -64,20 +64,18 @@ def hyperopt_results():
     hyperopt_configs = {
         "parameters": {
             "trainer.learning_rate": {
-                "type": "float",
-                "low": 0.0001,
-                "high": 0.01,
-                "space": "log",
-                "steps": 3,
```

Comment:
it there no steps in raytune loguniform?

Reply:
or maybe that's why qloguniform exists? (I have not looked deeply in raytune yet)

Reply:
@w4nderlust  converted to use `loguniform`.   After creating the PR, I realized there were a few other unit tests (`test_api.py` and `test_cli.py`) that invoked hyperopt functions.  Your observation was fixed to get these other unit tests to work.